corehq/apps/data_dictionary/util.py

```diff
@@ -303,3 +304,41 @@ def get_gps_properties(domain, case_type):
         case_type__name=case_type,
         data_type=CaseProperty.DataType.GPS,
     ).values_list('name', flat=True))
+
+
+def get_column_headings(row, valid_values, case_type=None):
+    column_headings = []
+    errors = []
+    for index, cell in enumerate(row, start=1):
+        if not cell.value:
+            if case_type:
+                errors.append(
+                    _("Column {} in case type {} has an empty header").format(index, case_type)
```

Comment:
Nit: I think it might be nice to enclose the case type in quotes?

Reply:
Thanks for the suggestion. I have replaced all instances of the case type string in quotes.